src/plugins/react/reactquestiondate.tsx

```diff
@@ -24,10 +24,14 @@ export default class SurveyQuestionDate extends React.Component<any, any> {
     }
     render(): JSX.Element {
         if (!this.question) return null;
-        var date = this.question.value ? moment(this.question.value, "DD/MM/YYYY") : moment();
+        var value = this.question.value;
         return (
             <div id={this.getDivId()}>
-                <DatePicker className="form-control" showYearDropdown selected={date} onChange={this.handleOnChange} />
+                <Cleave placeholder="DD/MM/YYYY"
+                className="form-control"
```

Comment:
My guess is that everyone tested on desktop, found the date picker to be weird, and then as soon as someone tries it on mobile they'll complain that they have to type the date in instead of having a date picker :P